src/components/Footer.vue

```diff
@@ -1,61 +1,107 @@
 <template>
-  <div class="footer">
-    <div class="social-network">
-      <ul>
-        <li>
-            <h1>Connected with us on social networks!</h1>
-        </li>
-        <li>
-          <a href=""><img alt="linkedin" src="../assets/linkedin.svg" /></a>
-        </li>
-        <li>
-          <a href=""><img alt="twitter" src="../assets/twitter.svg" /></a>
-        </li>
-        <li>
-          <a href=""><img alt="instagram" src="../assets/instagram.svg" /></a>
-        </li>
-        <li>
-          <a href=""><img alt="facebook" src="../assets/facebook.svg" /></a>
-        </li>
-      </ul>
+  <footer class="footer">
+    <div class="footer_container">
```

Comment:
I am using _ as a kind of secondary name class. But I will change the name of them to more descriptive names when I will create the new footer.

Reply:
What is a secondary name class?

Reply:
By secondary name of class I mean a second name for the class in order to give more details about it. For example footer_nav_list, the list is something like a secondary part of the name. Let me know if this isn't clear. Maybe I should change the way that I am naming the classes, if this makes confusion. 😏

Reply:
Hmmm yes it is not very intuitive. Naming is one of the hardest things. I think every class should be named after what it is supposed to do, in a way that is self-explanatory what this class does. I am very confused with the - and _. Intuitively it feels wrong to be inconsistent in the naming of variables and classes. Of course, there might be a web development naming pattern that I am not aware of :) @kkarayannis any idea?

Reply:
Just to see what other people do I looked at some open source projects listed [here](https://www.devaradise.com/vue-example-projects). Now, it could be that I didn’t look hard enough, but not only did I not find any mixed `-` and `_` uses, but I didn’t see any `_` used at all. Everyone seems to just be using `-` everywhere. Other than that, the names are ok. `footer-nav-list` and `footer-nav-item` are descriptive enough IMO.

Reply:
I have found that there is a css name convention called BEM (block, element, modifier). You can look it up, there are specific rules for it :) [->](http://getbem.com/introduction/)

Reply:
Ah thank you @eleniaslanidou89 ! This article explains very well the BEM architecture. If you like it we can use it in the whole project. In the screenshot above the naming is very similar to the one in the article, but the naming in the `Footer.vue` seems a bit different. Should we follow the convention stated in the article in the whole project? What do you think?

Reply:
Yes, we will follow it in the whole project :) I will change the names in the Footer.vue too.